src/js/user-profile/components/FormList.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import FormItem from './FormItem';
+
+class FormList extends React.Component {
+  render() {
+    const toggleModal = this.props.toggleModal;
```

Comment:
You can use destructuring here: `const { toggleModal, forms: savedForms } = this.props`

Reply:
Thanks, change made.